client/core/core_test.go

```diff
@@ -7558,7 +7539,8 @@ func TestPreimageSync(t *testing.T) {
 		lo := convertMsgLimitOrder(msgOrder)
 		resp := orderResponse(msg.ID, msgOrder, lo, false, false, false)
 		limitRouteProcessing <- lo.ID()
-		<-time.NewTimer(time.Millisecond * 100).C
+		commitSig <- lo.Commit
```

Comment:
I think because of the sequencing provided by the `errChan` that you can skip the channel and just  directly set a 
```Go
var commit order.Commitment
```